static/app/views/issueDetails/groupEventHeader.tsx

```diff
@@ -18,10 +22,27 @@ type GroupEventHeaderProps = {
 function GroupEventHeader({event, group, project}: GroupEventHeaderProps) {
   const organization = useOrganization();
   const issueTypeConfig = getConfigForIssueType(group, group.project);
+  const isRelatedIssuesEnabled = organization.features.includes(
+    'related-issues-issue-details-page'
+  );
+  // This is also called within the TraceTimeline component but caching will save a second call
+  const {isError, isLoading, oneOtherIssueEvent} = useTraceTimelineEvents({
+    event,
+  });
+  const readyToShow = !isLoading && !isError;
 
   return (
     <StyledDataSection>
       <GroupEventCarousel group={group} event={event} projectSlug={project.slug} />
+      {isRelatedIssuesEnabled && readyToShow && oneOtherIssueEvent === undefined && (
```

Comment:
When `oneOtherIssueEvent === undefined` is true, we have the normal case where we want to just display the tracelink and potentially the timeline.